index.html

```diff
@@ -821,7 +827,7 @@ <h2><code>application/vc+ld+sd-jwt</code></h2>
   
 
   <section id="json-web-token-claim-registrations">
-    <h3>JSON Web Token Claims</h3>
+    <h3>Registered Claims</h3>
```

Comment:
Adjusted this language since sd-jwt is not JWT but uses JWT registered claim names.